packages/compass/src/main/application.ts

```diff
@@ -312,6 +312,7 @@ class CompassApplication {
         '*://cloud.mongodb.com/*',
         '*://cloud-dev.mongodb.com/*',
         '*://compass.mongodb.com/*',
+        '*://localhost/*',
```

Comment:
probably only for dev mode?

Reply:
but now that i am think about it, do we want to run (prod) compass with local atlas? if not, then we should only add this URL for development purposes.

Reply:
Aligned with the injectCSP logic where we add localhost dynamically.